File: src/test/java/com/refactoring/java/FirstBasePriceTest.java
package com.refactoring.java;

import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.DisplayName;
import org.junit.jupiter.api.Test;

public class FirstBasePriceTest {

    @Test
    @DisplayName("1000")
    void getPriceTest() {
        com.refactoring.java.FirstBasePrice firstBasePrice = new FirstBasePrice(1000, 2);
        double result = firstBasePrice.getPrice();
        Assertions.assertEquals(result, 1900);
    }
}

File: src/test/kotlin/com/refactoring/kt/FirstBasePriceTest.kt
package com.refactoring.kt

import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

class FirstBasePriceTest {

    @Test
    internal fun `basePrice 1000 이상일때`() {
        val firstBasePrice = FirstBasePrice(1000, 2)
        var restult = firstBasePrice.getPrice()
        Assertions.assertEquals(restult, 1900.0)
    }

//    @Test
//    internal fun `basePrice 1000 이하일때`() {
//        val firstBasePrice = FirstBasePrice(10, 2)
//        var restult = firstBasePrice.getPrice()
//        Assertions.assertEquals(restult, 19.6)
//    }
}